components/borrow.tsx

```diff
@@ -273,66 +296,46 @@ const Borrow = ({
               <div className="text-gray-500">Total Borrow Amount</div>
               <div className="font-bold">
                 $
-                {positions.reduce(
-                  (acc, position) => +position.price * +position.borrowed + acc,
-                  0,
-                )}
+                {positions
+                  .reduce(
+                    (acc, { underlying, amount }) =>
+                      +formatUnits(amount, underlying.decimals) *
```

Comment:
Let's use the `dollarValue` function to do the math here.

Reply:
https://github.com/clober-dex/coupon.finance/pull/77/commits/4545c84ab98b9e4f5a2bced9147d9e6c83a3a060